camunda-formio-tasklist-vue/src/services/SocketIOServices.ts

```diff
@@ -1,9 +1,14 @@
-import { engine, socketUrl } from "../services/constants";
+import { engine, socketUrl } from "./constants";
 import AES from "crypto-js/aes";
 import SockJS from "sockjs-client";
 import Stomp from "stompjs";
-
+/* eslint-disable */
```

Comment:
I'd recommend removing eslint disable. If needed, you can just disable the specific rule to disable